---a/net/minecraft/block/BlockDirectional.java
+++b/net/minecraft/block/BlockDirectional.java
@@ -4,14 +4,13 @@
 import net.minecraft.block.properties.PropertyDirection;
 import net.minecraft.util.EnumFacing;
 
-public abstract class BlockDirectional extends Block
-{
+public abstract class BlockDirectional extends Block {
+
     public static final PropertyDirection FACING = PropertyDirection.create("facing", EnumFacing.Plane.HORIZONTAL);
     private static final String __OBFID = "CL_00000227";
 
-    protected BlockDirectional(Material materialIn)
-    {
+    protected BlockDirectional(Material materialIn) {
         super(materialIn);
     }
-}
+}
